contracts/contracts/coordination/Coordinator.sol

```diff
@@ -252,9 +251,6 @@ contract Coordinator is AccessControlDefaultAdminRules {
         return id;
     }
 
-<<<<<<< HEAD
-    function cohortFingerprint(address[] calldata nodes) public pure returns (bytes32) {
-=======
```

Comment:
Looks like a merge conflict made it's way into the diff here.

Reply:
seems outdated? i can't find that